industry/products.md

```diff
@@ -26,39 +26,62 @@ Raw models usually do not include the features included in many production syste
 - HTML/XML tag handling
 
 
-| Company | Product | Self-serve |
-| --- | --- | :-: |
-| Google | Google Translate | &#10004; |
-| [LingvaNex](/../industry/companies.md#lingvanex) | Translation API | &#10004; |
-| Yandex | Translate API | &#10004; |
-| Microsoft | Translator Text API | &#10004; |
-| Amazon | Translate API | &#10004; |
-| [ModernMT](/../industry/companies.md#modernmt) | API | &#10004; |
-| IBM | Watson Language Translator | &#10004; |
-| Baidu | 翻译API | &#10004; |
-| [DeepL](/../industry/companies.md#deepl) | API | &#10004; |
-| Alibaba | Machine Translation | &#10004; |
-| [Systran](/../industry/companies.md#systran) | Systran Translate Pro | &#10004; |
-| [Unbabel](/../industry/companies.md#unbabel) | Unbabel Translate | |
-| [Lilt](/../industry/companies.md#lilt) | Lilt Translate | |
-| [Tilde](/../industry/companies.md#tilde) | TildeMT | |
-| [Omniscien](/../industry/companies.md#omniscien-technologies) | Language Studio | |
-| [KantanMT](/../industry/companies.md#kantanmt) | Translate AI | |
-| [Pangeanic](/../industry/companies.md#pangeanic) | PangeaMT | |
-| TextShuttle | TextShuttle AI Translation | |
-| Globalese | Globalese | |
-| RWS Group | Language Weaver (Iconic) | &#10004; |
-| Mirai Translate | Mirai Translator | &#10004; |
-
+| Product | Self-serve |
+| --- | --- |
+| NiuTrans API by **[Xiaoniu Fanyi](/../industry/companies.md#xiaoniu-fanyi)** | ✔ |
+| [KantanMT](/../industry/companies.md#kantanmt) by **Keywords Studios** | ✔ |
+| **Google** Translate | ✔ |
+| **Microsoft** Translator Text API | ✔ |
+| **[LingvaNex](/../industry/companies.md#lingvanex)** Translator | ✔ |
+| **[Language I/O](/../industry/companies.md#language-io)** Translation API | ✔ |
+| **Yandex** Translate | ✔ |
+| **BytePlus** Translate | ✔ |
+| **Amazon** Translate | ✔ |
+| Xinyi by **[Zhongke Fanyu Technology](/../industry/companies.md#zhongke-fanyu-technology)** | ✔ |
```

Comment:
I don't love the *{ product name } by { company name }* here and throughout, I would just do *{ company name } { product name }*.